docs/BR.md

```diff
@@ -1272,55 +1279,64 @@ No stipulation.
 
 **Note**: Following certificate issuance, a certificate may be revoked for reasons stated in [Section 4.9](#49-certificate-revocation-and-suspension). Therefore, relying parties should check the revocation status of all certificates that contain a CDP or OCSP pointer.
 
-### 4.9.7 CRL issuance frequency (if applicable)
-
-For the status of Subscriber Certificates:
-
-If the CA publishes a CRL, then the CA SHALL update and reissue CRLs at least once every seven days, and the value of the `nextUpdate` field MUST NOT be more than ten days beyond the value of the `thisUpdate` field.
+### 4.9.7 CRL issuance frequency
 
-For the status of Subordinate CA Certificates:
+CAs MUST generate and publish either:
+- a full and complete; OR 
+- partitioned (i.e., "sharded") CRLs, that when aggregated, represent the equivalent of the full and complete CRL.
 
-The CA SHALL update and reissue CRLs at least:
+CRLs must be available via a publicly-accessible HTTP URL. 
 
-  i. once every twelve months; and
-  ii. within 24 hours after revoking a Subordinate CA Certificate.
+CAs issuing Subscriber Certificates (i.e., Certificates for servers) SHALL:
+- update and reissue CRLs at least once daily; and
```

Comment:
If the intent is indeed to move from "at least once every seven days", to "at least daily", should we rephrase this to "at least once every 24 hours"?